var/spack/repos/builtin/packages/glvis/package.py

```diff
@@ -0,0 +1,99 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Glvis(Package):
+    """GLVis: an OpenGL tool for visualization of FEM meshes and functions"""
+
+    homepage = "http://glvis.org/"
+    url      = "https://github.com/GLvis/glvis/archive/v3.3.tar.gz"
```

Comment:
Should this be `https://github.com/glvis/glvis`?

Reply:
Package versions are usually parsed from the `url`, if there are tarballs, so in the current version, the `url` is correct. You are correct that after I make the changes you propose, the `url` should be `https://github.com/glvis/glvis`, because GitHub repos are typically used for "GitHub only" packages, which is more or less what the `mfem` spack package looks like.